src/Kestrel.Core/Internal/Infrastructure/ConnectionReference.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Connections;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure
+{
+    public class ConnectionReference
```

Comment:
I'm surprised git didn't treat this as a move. Anything change other than HttpConnection -> KestrelConnection?

Reply:
Nope.